openwisp_controller/config/static/config/css/admin.css

```diff
@@ -257,7 +285,6 @@ div.tabs-divider{
   ul.tabs a:hover,
   ul.tabs a:focus {
       display: block;
-      border-radius 0;
```

Comment:
Thanks for this, I wonder why our QA checks were not spotting this :rage:

Reply:
I have checked `run-qa-checks`. I think stylelint checks are not present.

Reply:
Thanks, you're right, created an issue for it